go.mod

```diff
@@ -6,19 +6,23 @@ require (
 	github.com/NYTimes/gziphandler v0.0.0-20170623195520-56545f4a5d46
 	github.com/PuerkitoBio/purell v1.0.0 // indirect
 	github.com/PuerkitoBio/urlesc v0.0.0-20160726150825-5bd2802263f2 // indirect
-	github.com/davecgh/go-spew v1.1.0
+	github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a
```

Comment:
@sttts Do we want to stick to versions we already use in k/k?

```github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a
github.com/davecgh/go-spew v1.1.1
github.com/go-openapi/jsonpointer v0.19.3
github.com/go-openapi/jsonreference v0.19.3
github.com/go-openapi/swag v0.19.5
github.com/google/uuid v1.1.1
github.com/mitchellh/mapstructure v1.1.2```

Reply:
yes, makes sense. Let me fix that

Reply:
done